### PR TITLE
Code commit for DP-1.4

### PR DESCRIPTION
**Pull request by azubaidhar**
Added functions to configure OC-QOS forwarding-classes,  queues, classifiers, attach classifier to interface so expected number of queues/packets are seen.

Thanks,
Abdul Zubaidhar .A

### CODE REVIEW

File: feature/experimental/qos/ate_tests/qos_output_queue_counters_test/qos_output_queue_counters_test.go [hunk 1]
```diff
@@ -60,6 +60,165 @@ func TestMain(m *testing.M) {
...
+		desc:         "forwarding-group-AF4",
+		queueName:    "2",
+		targetGrpoup: "target-group-AF4",
+	}, {
+		desc:         "forwarding-group-NC1",
+		queueName:    "3",
+		targetGrpoup: "target-group-NC1",
+	}}
+
+	t.Logf("qos forwarding groups config cases: %v", cases)
```
Comment (greg-dennis) [inline feature/experimental/qos/ate_tests/qos_output_queue_counters_test/qos_output_queue_counters_test.go:98]: Log lines should be capitalized

**Review by greg-dennis — APPROVED** [review]
Approving for Go only

**Comment by sezhang2** [comment]
I updated the test and added the config. Can you pick the latest code?

**Review by sezhang2 — CHANGES_REQUESTED** [review]
Please pick the latest code.

**Comment by sgirishj** [comment]
PR is outdated due to https://github.com/openconfig/featureprofiles/pull/1018

Hence closing.

**Pull request closed without merging**